packages/render-fidelity-tools/src/components/renderers/gltfpt-viewer.ts

```diff
@@ -77,41 +76,35 @@ export class PathtracingViewer extends LitElement {
   private async[$updateScenario](scenario: ScenarioConfig) {
     this[$canvas] = this.shadowRoot!.querySelector('canvas');
 
-    const dpr = resolveDpr();
-
-    const settings = {
-      pathtracing: true,
-      debugMode: 'None',
-      maxBounceDepth: 1,
-      useIBL: true,
-      autoScaleOnImport: false,
-      pixelRatio: 1.0 / dpr,  // TODO: clarify
-      autoRotate: false,
-      disableDirectShadows: true
-    };
-
-    this[$renderer] =
-        new PathtracingRenderer({'canvas': this[$canvas]!}, settings);
+    const enableControls = false;
+    this[$renderer] = new PathtracingRenderer(this[$canvas]!, enableControls);
 
     const renderer = this[$renderer];
+    renderer.autoScaleOnImport(false);
+    renderer.setPixelRatio(0.5);
+    renderer.setMaxBounceDepth(8);
```

Comment:
This seems like a lot, but you know path tracers better than I do. Have you found particular sample models where you need this many bounces to get good convergence?

Reply:
The reason why I set this to 8 is realted to this issue
https://github.com/DassaultSystemes-Technology/dspbr-pt/issues/6

In general, having a low bounce depth usually intrduces error compared to a fully unbiased integration method, e.g. with probabilistic ray-termination or just very high bounce depth. 
But you're right, with the current scenarios, with the issue linked above being resolved, less bounces should be enough to capture relevant features with an acceptablel error.

The question which arises when thinking about rasterizer validation with pathtracing is "how much global illumination features do actually makes senses to include?".  I mentioned this before with thoughts on artificially excluding some of the full light transport features, e.g. all indirect diffuse bounce light > 1 (that's were most rasterizers will stop faking gi), or no direct shadowing from IBL, ...

Reply:
Thanks for the info; sounds like we should reevaluate when that bug is fixed. As for exclusion, I'm not sure, but I'm generally against it. I don't expect the rasterizers to ever attain the quality of the path tracer, but I'd rather see the errors than hide them. I prefer these tests to demonstrate the known shortcomings of our rasterizers, and make it known that these differences are places where users should expect rendering changes to occur over time as we build more features in and improve.